frontend/ui/src/actions/conversations/index.ts

```diff
@@ -87,7 +87,7 @@ export const getConversationInfo = (conversationId: string, retries?: number) =>
     });
 
 export const readConversations = (conversationId: string) => (dispatch: Dispatch<any>) => {
-  HttpClientInstance.readConversations(conversationId).then(() =>
+  HttpClientInstance.markConversationAsRead(conversationId).then(() =>
```

Comment:
Having the same name as the endpoint was intended. I think it's good for consistency. I would like to keep it that way. What do you think @lucapette @airyhq/codeowners-frontend ?

Reply:
The method name looked not right for me. More a "get a list of conversations" and not "mark this one conversation as read"

Reply:
I agree it doesn't look right but I would also not open the door for diverging the api methods names from the actual endpoints (the convention is "reverse the namespace") right now. 

So what I think we should do is keeping the convention and rediscuss the naming of `/conversations.read` (because I would argue that's the wrong name?)

Reply:
Yeah, I agree. The problem is the name of the endpoint, maybe `/conversations.markAsRead` or maybe `/conversations.markAs.read` and in the future we can have more endpoints like `/conversations.markAs.XXXX`

Reply:
or without the `markAs`, just `mark`

Reply:
https://github.com/airyhq/airy/issues/1211 created a ticket for it, reverted the change here for now.